src/components/swap/ConfirmSwapModal.tsx

```diff
@@ -58,9 +58,13 @@ const StyledL2Logo = styled.img`
   width: 16px;
 `
 
+// Any existing USDT or LDO allowance needs to be reset before we can approve the new amount (mainnet only).
+// See the `approve` function here: https://etherscan.io/address/0xdAC17F958D2ee523a2206206994597C13D831ec7#code
+const RESET_APPROVAL_TOKENS = [USDT_MAINNET, LDO]
```

Comment:
should this be in a constants file instead?

Reply:
it should